docs/_includes/usage/cf-layout/usage.md

```diff
@@ -1290,208 +1290,181 @@ height of the next parent using `position: relative;`. This means that the
-                instructior ex pri. Cu pri inani constituto, cum aeque noster
-                commodo cu.
-            </p>
-            <a class="a-link
-                      a-link__jump
-                      a-link__icon-after-text">
-                <span class="a-link_text">Read more about the feature</span>
-                {% raw %}{% include icons/right.svg %}{% endraw %}
-            </a>
-        </div>
-        <div class="o-featured-content-module_visual">
-            <img class="o-featured-content-module_img"
-                 src="http://placekitten.com/g/540/270">
-        </div>
-    </section>
-</div>
+<section class="o-featured-content-module">
+    <div class="o-featured-content-module_text">
+        <h2>Featured content</h2>
+        <p>
+            Lorem ipsum dolor sit amet, ei ius adhuc inani iudico, labitur
+            instructior ex pri. Cu pri inani constituto, cum aeque noster
+            commodo cu.
+        </p>
+        <a class="a-link
+                  a-link__jump
+                  a-link__icon-after-text">
+            <span class="a-link_text">Read more about the feature</span>
+            {% raw %}{% include icons/right.svg %}{% endraw %}
+        </a>
+    </div>
+    <div class="o-featured-content-module_visual">
+        <img class="o-featured-content-module_img"
+             src="http://placekitten.com/g/540/270"
+             alt="">
+    </div>
+</section>
 ```
 
-### Featured content module - Maps
-
-When the featured content module image is a map (or other right-aligned
-content), the `o-featured-content-module__right` modifier class is added to
-the `o-featured-content-module` organism. This anchors the image to the right
-side so that the copyright information is displayed.
-
-<div class="block block__border block__flush-sides">
-    <section class="o-featured-content-module
-                    o-featured-content-module__right">
-        <div class="o-featured-content-module_text">
-            <div class="a-heading a-heading__icon">
-                {% include icons/speech-bubble.svg %}
-                Featured map
-            </div>
-            <h2>Feature title</h2>
-            <p>
-                Lorem ipsum dolor sit amet, ei ius adhuc inani iudico, labitur
-                instructior ex pri. Cu pri inani constituto, cum aeque noster
-                commodo cu.
-            </p>
-            <a class="a-link
-                      a-link__jump
-                      a-link__icon-after-text">
-                <span class="a-link_text">Read more about the feature</span>
-                {% include icons/right.svg %}
-            </a>
-        </div>
-        <div class="o-featured-content-module_visual">
-            <img class="o-featured-content-module_img"
-                 src="http://placekitten.com/g/540/270">
-        </div>
-    </section>
-</div>
+### Anchoring the visual on the right
+
+When one wants to anchor a featured content module's visual on the right
```

Comment:
Could be shortened to `To anchor a featured content module's visual on the right…`

Reply:
:+1: Updated in 43458fae.